### ISSUE TITLE
Date pickers default to US date format: Change to use @material-ui/pickers library

### ISSUE THREAD

**Issue by ramdesh** [ECLK/IncidentManagement#390]
**Is your feature request related to a problem? Please describe.**
The current (native) date pickers default to the US `mm/dd/yyyy `date format, whereas Sri Lanka generally uses `dd/mm/yyyy` or `yyyy/mm/dd`. 

**Describe the solution you'd like**
With the use of `@material-ui/pickers` instead of native pickers, we can set the date format that we need (native pickers do not allow this according to [this thread](https://github.com/mui-org/material-ui/issues/10251)). 

**Describe alternatives you've considered**
Using `@material-ui/pickers`, we can easily set the date format using the `dateFormat` attribute.

**Comment by MrClemRkz** [comment]
As of LIFe, government standard for 'Date of Birth' is `yyyy-mm-dd`. So I guess we could stick with this format as the conventional for date format. Will have this conventions set on documentation.

Ref link: http://life.gov.lk/index.php?option=com_standards&view=standards&Itemid=141&lang=en

**Comment by BandulaRanathunga** [comment]
> As of LIFe, government standard for 'Date of Birth' is `yyyy-mm-dd`. So I guess we could stick with this format as the conventional for date format. Will have this conventions set on documentation.
> 
> Ref link: http://life.gov.lk/index.php?option=com_standards&view=standards&Itemid=141&lang=en

Yes, it should be yyyy-mm-dd. There was a very rich discussion on this matter in LIFe. Please be strict to this in everywhere of the system.

**Comment by ramdesh** [comment]
Great, I'll work on a PR to use `@material-ui/pickers` and default the date format to `yyyy-mm-dd`. Thanks for the input!

**Comment by MrClemRkz** [comment]
estimation: 6hrs
